src/sentry/api/endpoints/organization_member_details.py

```diff
@@ -97,6 +101,20 @@ def _is_only_owner(self, member):
 
         return True
 
+    def _serialize_member(self, member, request, allowed_roles=None):
+        context = serialize(
+            member,
+            serializer=OrganizationMemberWithTeamsSerializer()
+        )
+
+        if request.access.has_scope('member:admin'):
+            context['invite_link'] = member.get_invite_link()
```

Comment:
this seems correct to me, but also I think it warrants and extra security look from matt/ted/jess/cramer

Reply:
seems right to me, but we can ask matt for a quick review to make sure

Reply:
Why do we need to return an individualized invite link to someone the link isn't for? I don't think this is a good idea to return this to other existing members.

Reply:
@mattrobenolt This is the behavior that's in prod right now, this is from cramer from slack:
```
we should keep it for now
id like to add a generic invite link at some point
but the invite link is like “email isnt working” or “i want to send this myself instead of via sentry”
```

Reply:
Oh. TIL. I thought we only had a CTA to trigger sending a new email to them.